### ISSUE TITLE
Xcode 14 stack

### ISSUE THREAD

**Issue by Oupsla** [bitrise-steplib/steps-deploy-to-itunesconnect-deliver#123]
### Troubleshooting

- [x] I've searched [discuss.bitrise.io](https://discuss.bitrise.io/) for possible solutions.
- Which version of the step is effected? 4.6.0
- Is the issue reproducible with the latest version? YES
- Does the issue happen sporadically, or every time? EVERY TIME
- Is the issue reproducible locally by following our [local debug guide](https://devcenter.bitrise.io/troubleshooting/debugging-your-build-on-your-own-machine/)? YES / NO / NOT APPLICABLE

### Issue description

When using stack with xCode 14.0.x, the step fails automatically with the following message : 
```
Could not find transporter at /Applications/Xcode-14.Release.Candidate.app/Contents/Developer/. Please make sure you set the correct path to your Xcode installation
```

**Comment by airdrummingfool** [comment]
I believe this was just fixed by Fastlane in [2.210.0](https://github.com/fastlane/fastlane/releases/tag/2.210.0) via https://github.com/fastlane/fastlane/pull/20631 . Ref https://github.com/fastlane/fastlane/issues/20371

**Comment by Oupsla** [comment]
Resolved on our side too 😄
